test/integration/framework/applications/guestbooktest.go

```diff
@@ -110,25 +105,22 @@ func (t *GuestBookTest) DeployGuestBookApp(ctx context.Context, gardenerTestOper
 	gomega.Expect(err).NotTo(gomega.HaveOccurred())
 
 	ginkgo.By("Applying redis chart")
+	// redis-slaves are not required for test success
+	chartOverrides := map[string]interface{}{
+		"cluster": map[string]interface{}{
+			"enabled": false,
+			"slaveCount": 0,
```

Comment:
Can we skip specifying this as `slaveCount ` is the replicas of the slave sts, but we have `enabled: false` which means that no slave sts is created.
```suggestion
			"slaveCount": 0,
```

Reply:
done